src/client/pages/CountryHome/Repository/Panel/Actions/Actions.tsx

```diff
@@ -3,31 +3,43 @@ import React from 'react'
 import { useTranslation } from 'react-i18next'
 
 import classNames from 'classnames'
+import { Objects } from 'utils/objects'
 
-import { useIsRepositoryLoading } from 'client/store/ui/repository/hooks'
+import { useIsRepositoryLoading, useRepositoryItem } from 'client/store/ui/repository/hooks'
+import { useClosePanel } from 'client/pages/CountryHome/Repository/hooks/useClosePanel'
 
-type Props = {
-  setOpenPanel: (open: boolean) => void
-  onSave: () => void
-}
+import { useOnSaveFile } from './hooks/useOnSaveFile'
 
-const Actions: React.FC<Props> = (props: Props) => {
-  const { setOpenPanel, onSave } = props
+const Actions: React.FC = () => {
   const { t } = useTranslation()
+  const repositoryItem = useRepositoryItem()
+
+  const onSaveFile = useOnSaveFile()
 
+  // TODO: Implement this
+  const onUpdateFile = () => {}
```

Comment:
TODO:
Implement and rename to useOnSaveRepositoryItem, also to handle the file upload here.

**Do not save the file to redux store!**